checker/manual/external-tools.tex

```diff
@@ -792,6 +793,24 @@ \section{tIDE\label{tide}}
 % See its documentation at \myurl{http://tide.olympe.in/}.
 \end{sloppypar}
 
+\section{NetBeans\label{netbeans}}
+To run a checker within NetBeans:
+
+\begin{enumerate}
+\item Add the checker framework libraries to your project's library. To do so, 
+right click on the project in the Projects panel, and select "Properties" in the
+drop-down menu. Then, in the "Project Properties" window, navigate to the "Libraries"
+tab and click the "Add JAR/Folder" button on the right, browse and select 
+\<CHECKERFRAMEWORK/checker/dist/checker.jar>.
+\item
+Enable annotation processor underlining in the editor. Go to "Build>Compiling" and check
+the box "Enable Annotation Processing," and under that, "Enable Annotation Processing in Editor."
+\item
+Add the checker to run, by clicking "Add" next to the box labeled "Annotation Processors" and enter
+the fully qualified name of the checker (for example, \<org.checkerframework.checker.nullness.NullnessChecker>)
+and click "OK" to add.
+\end{enumerate}
+The selected checker should be run on the project either on a save (if Compile on Save is enabled), or when the project is built, and annotations will appear in the editor. 
```

Comment:
Instead of "annotations will appear in the editor" should this be "processor output..." ? Or are these "IDE annotations" in contrast to the Java annotations we usually talk about in the document?
(Wraping the long line would also be good.)